app/kumactl/cmd/uninstall/uninstall_ebpf.go

```diff
@@ -221,8 +221,7 @@ func (r *CleanupJob) getPodLogs(
 ) (string, error) {
 	podLogs, err := r.podClient.GetLogs(
 		pod.Name,
-		&corev1.PodLogOptions{},
-	).Stream(ctx)
+		&corev1.PodLogOptions{}).Stream(ctx)
```

Comment:
Is this coming from gofumpt or is this something needed to change manually? I much prefer how it was before for reading :thinking:

Reply:
It's a manual change. I did replace `\},\n\s*\)`with `})`

Reply:
Could we do something like `\},\n\s*\)\n` -> `})\n` instead?

Reply:
Would you like to apply this from n-1 commit ?

Reply:
To whatever commit gave us:

```
},
)
```

as Bart pointed out, I suppose.

The expected rule is something like different indentations remain on their own lines but there are no "unnecessary" new lines. 

Here we have two indentations:

```
    &corev1.PodLogOptions{},
```

is one and

```
).Stream(ctx)
```

is another. But the original case pointed out was:

```
		rootCmd.SetArgs([]string{
                   ...
		},
		)
```

with the opening `(` and `{` appearing on the same indentation, and so their closing braces should too.

Reply:
BTW if you prefer I think we should get a pure `gofumpt` PR merged and then tweak later on in other PRs.

Reply:
Ok, let's do this , I rollback the extra commits

Reply:
It's done ;) !

Reply:
BTW, I created [an issue at gofumpt](https://github.com/mvdan/gofumpt/issues/262), if you want to contribute to the discussion

Reply:
I added a commit to fix just that file since that seems like only place or one of the very few places with that style.